_source/user-guide/infrastructure-monitoring/jenkins.md

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
When I open the preview of this [page](https://deploy-preview-1615--logz-docs.netlify.app/user-guide/infrastructure-monitoring/metrics-dashboards/jenkins.html) it looks weird at the bottom. 
Is it just cause it's a preview mode?